Snowflake.Data.Tests/UnitTests/SFSessionPropertyTest.cs

```diff
@@ -26,12 +26,35 @@ public void TestThatPropertiesAreParsed(TestCase testcase)
             CollectionAssert.AreEquivalent(testcase.ExpectedProperties, properties);
         }
 
+        [Test]
+        [TestCase("a", "a", "a.snowflakecomputing.com")]
```

Comment:
lets use real case scenarios from the docs:
https://docs.snowflake.com/en/user-guide/admin-account-identifier

Reply:
added an example from doc